packages/zero-runtime/README.md

```diff
@@ -249,3 +255,202 @@ const ExtraHeading = styled(Heading)({
+
+```jsx
+const theme = extendTheme({
+  colors: {
+    primary: 'tomato',
+    secondary: 'cyan',
+  },
+});
+
+console.log(theme.colors.primary); // 'tomato'
+console.log(theme.vars.colors.primary); // 'var(--colors-primary)'
+```
+
+#### Color schemes
+
+Some tokens, especially color-related tokens, can have different values for different scenarios. For example in a daylight condition, the background color might be white, but in a dark condition, it might be black.
+
+The `extendTheme` utility lets you define theme with a special `colorSchemes` key:
+
+```jsx
+extendTheme({
+  colorSchemes: {
+    light: {
+      colors: {
+        background: '#f9f9f9',
+        foreground: '#121212',
+      },
+    },
+    dark: {
+      colors: {
+        background: '#212121',
+        foreground: '#fff',
+      },
+    },
+  },
+});
+```
+
+In the above example, `light` (default) and `dark` color schemes are defined. The structure of each color scheme must be a plain object with keys and values.
+
+#### Switching color schemes
+
+By default, when `colorSchemes` is defined, zero-runtime uses the [`prefers-color-scheme`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) media query to switch between color schemes based on user's system settings.
+
+However, if you want to control the color scheme based on application logic, for example, using a button to switch between light and dark mode, you can customize the behavior by providing a `getSelector` function:
+
+```diff
+  extendTheme({
+    colorSchemes: {
+      light: { ... },
+      dark: { ... },
+    },
++   getSelector: (colorScheme) => colorScheme ? `.theme-${colorScheme}` : ':root',
+  });
+```
+
+Note that you need to add a logic to a button by yourself. Here is an example of how to do it:
+
+```jsx
+function App() {
+  return (
+    <button
+      onClick={() => {
+        document.documentElement.classList.toggle('theme-dark');
+      }}
+    >
+      Toggle color scheme
+    </button>
+  );
+}
+```
+
+#### CSS variables prefix
+
+You can add a prefix to the generated CSS variables by providing a `cssVarPrefix` option to the `extendTheme` utility:
+
+```jsx
+extendTheme({
+  cssVarPrefix: 'zero',
+});
+```
+
+The generated CSS variables will have the `zero` prefix:
+
+```css
+:root {
+  --zero-colors-background: #f9f9f9;
+  --zero-colors-foreground: #121212;
+}
+```
+
+#### TypeScript
+
+To get the type checking for the theme, you need to augment the theme type:
+
+```ts
+// any file that is included in your tsconfig.json
+import type { ExtendTheme } from '@mui/zero-runtime';
+
+declare module '@mui/zero-runtime/theme' {
+  interface ThemeTokens {
+    // the structure of your theme
+  }
+
+  interface ThemeArgs {
+    theme: ExtendTheme<{
+      colorScheme: 'light' | 'dark';
+      tokens: ThemeTokens;
+    }>;
+  }
+}
+```
+
+#### Mutliple themes
+
+> ⏳ Currently, zero-runtime only supports a single theme.
```

Comment:
This is not a limitation right now. So wording is not correct. We do support multiple themes, either through `colorSchemes` or user can create their own directly using css variables. We just maybe need a utility to expose the variable generation. They could also do something like -

```js
import { css } from '@mui/zero-runtime';

const customTheme = css`
  --color-primary: blue;
  /* rest of the variables */
`;
```

and then apply the `theme` class to the parent most element.

Reply:
Thanks for the clarification. I think I will remove this section to not confuse the readers for now. We can add it later, I don't think it's the major use case right now.